helpdesk/settings.py

```diff
@@ -6,6 +6,7 @@
 from django import forms
 from django.conf import settings
 from django.core.exceptions import ImproperlyConfigured
+from django.utils.translation import gettext, gettext_lazy as _
```

Comment:
You have an unused import which creates an error in the CI

```suggestion
from django.utils.translation import gettext_lazy as _
```

Reply:
Thanks for the quick feedback and good catch for the import.

I will update the docs as requested.